google/cloud/firestore_v1/document.py

```diff
@@ -389,11 +389,12 @@ def get(
                 :attr:`create_time` attributes will all be ``None`` and
                 its :attr:`exists` attribute will be ``False``.
         """
-        request, kwargs = self._prep_get(field_paths, transaction, retry, timeout)
+        request, kwargs = self._prep_batch_get(field_paths, transaction, retry, timeout)
 
         firestore_api = self._client._firestore_api
+        read_time: Timestamp = None
         try:
-            document_pb = firestore_api.get_document(
+            batch_response_pb: BatchGetDocumentsResponse = firestore_api.batch_get_documents(
```

Comment:
a type seems possibly wrong. If this is a a BatchGetDocumentsResponse, why are we casting to a list and getting the first element on ln 406?

Reply:
Oh you're right - this is a StreamIterator or some such thing. Will change.